src/platforms/esp32/test/CMakeLists.txt

```diff
@@ -31,3 +31,8 @@ set(TEST_COMPONENTS "testable" CACHE STRING "List of components to test")
 
 include($ENV{IDF_PATH}/tools/cmake/project.cmake)
 project(atomvm-esp32-test)
+
+idf_build_get_property(c_compile_options C_COMPILE_OPTIONS)
+list(REMOVE_ITEM c_compile_options -std=gnu99)
+list(APPEND c_compile_options -std=gnu11)
+idf_build_set_property(C_COMPILE_OPTIONS ${c_compile_options})
```

Comment:
Just a warning, but I'm not sure yet where this is ok and how, the same set of CMake properties on the ESP32 root CMake file is causing issues when building esp-idf 5 that looks like it is using gnu17 as C standard.

Reply:
Latest version should take care of this.